### ISSUE TITLE
LogParser functionality is broken

### ISSUE THREAD

**Issue by Danlock** [awslabs/aws-waf-security-automations#46]
LogParser is supposed to have an REQUEST_PER_MINUTE_LIMIT environment variable that references the RequestThreshold parameter. Because it doesnt, the function get_outstanding_requesters never fully runs and the entire lambda is useless. 

Please merge the relevant pull request, or add the relevant code to your own cloudformation stack if you are a user.

**Comment by Danlock** [comment]
This might need to get changed in the cloudfront template as well, I havent looked at that one as I do not use cloudfront.

**Comment by Danlock** [comment]
LogParser blocks IPs once the env variable is added, but doesnt ever unblock them due to a python issue in comparing naive and timezone aware datetimes.

`can't subtract offset-naive and offset-aware datetimes `

Is the error message given.

**Comment by attobyte** [comment]
Is this issue really closed? The REQUEST_PER_MINUTE_LIMIT is still missing from v2.2. I am expericing the exact issue you detailed and your change for v2.1 has not made it into v2.2 or the latest repo. Figured I would reach before opening another issue incase I am missing something.

**Comment by hvital** [comment]
Hi @attobyte 

I added more details about the REQUEST_PER_MINUTE_LIMIT in #74 . If I'm not wrong, what we closed here was the timezone issue commented above. 

https://github.com/awslabs/aws-waf-security-automations/blob/c7b25b4f9005469f1606e8da19c9704268f0697a/source/log-parser/log-parser.py#L190
